pkg/action/build-uki.go

```diff
@@ -183,7 +183,7 @@ func (b *BuildUKIAction) createSystemdConf(sourceDir string) error {
 		finalEfiConf = nameFromCmdline(constants.UkiCmdline+" "+constants.UkiCmdlineInstall) + ".conf"
 	}
 	// Set that as default selection for booting
-	data := fmt.Sprintf("default %s\ntimeout 5\nconsole-mode max\neditor no\nsecure-boot-enroll manual\n", finalEfiConf)
+	data := fmt.Sprintf("default %s\ntimeout 5\nconsole-mode max\neditor no\nsecure-boot-enroll if-safe\n", finalEfiConf)
```

Comment:
I see the docs say this only works inside a vm. I wonder if this will not be problematic when users expect it to work also on hardware

https://www.freedesktop.org/software/systemd/man/latest/loader.conf.html

Reply:
ok found https://github.com/kairos-io/kairos/pull/2381#issuecomment-2015156480

Reply:
and the description of the pr also talked about it LOL sorry still a bit tired from all kubecon :P

Reply:
Well, unless users got here and look at the loader they should only know what we tell them.

We will need to make sure to make this very clear on our docs, that under a VM the workflow is automated vs in real hardware